### PR TITLE
Added Feast streaming features tutorial

### PR DESCRIPTION
**Pull request by yuce**

### CODE REVIEW

File: docs/modules/integrate/pages/streaming-features-with-feast.adoc [hunk 1]
```diff
@@ -0,0 +1,363 @@
+= Feast Streaming Features
+:description: This tutorial will get you started with streaming features using the Hazelcast integration with Feast.
+
+{description}
+
+== What You'll Learn
+
+You will setup an offline feature store with PostgreSQL and an online feature store with Hazelcast.
+Then, update the online feature store using a Jet job in real time from transctions streaming in a Kafka topic.
```
Comment (oliverhowell) [inline docs/modules/integrate/pages/streaming-features-with-feast.adoc:9]:
```suggestion
Then update the online feature store using a Jet job in real time from transactions streaming in a Kafka topic.
```

File: docs/modules/integrate/pages/streaming-features-with-feast.adoc [hunk 2]
```diff
@@ -0,0 +1,363 @@
...
+
+== Before you Begin
+
+You will need the following ready before starting the tutorial:
+
+* Hazelcast CLC. link:https://docs.hazelcast.com/clc/latest/install-clc[Installation instructions]
```
Comment (oliverhowell) [inline docs/modules/integrate/pages/streaming-features-with-feast.adoc:15]:
```suggestion
* Hazelcast CLC - link:https://docs.hazelcast.com/clc/latest/install-clc[Installation instructions]
```

File: docs/modules/integrate/pages/streaming-features-with-feast.adoc [hunk 3]
```diff
@@ -0,0 +1,363 @@
...
++
+[source,shell]
+----
+feast -c feature_repo serve -h 0.0.0.0 -p 6566 --no-access-log 2> /dev/null &
+----
++
+
+== Feature Transformation Using Jet
+
+You will create a Jet job tha reads transactions from a Kakfa topic, computes the number of transactions per user in the last seven-days buckets and populates the Hazelcast online store.
```
Comment (oliverhowell) [inline docs/modules/integrate/pages/streaming-features-with-feast.adoc:143]:
```suggestion
You will create a Jet job that reads transactions from a Kakfa topic, computes the number of transactions per user in the last seven-days' buckets and populates the Hazelcast online store.
```

File: docs/modules/integrate/pages/streaming-features-with-feast.adoc [hunk 4]
```diff
@@ -0,0 +1,363 @@
...
+        "2024-07-29T07:24:00Z"
+      ]
+    }
+  ]
+}
+----
+== Summary
+
+In this tutorial, you learned how to set up a feature engineering project that uses Hazelcast as the online store.
+You also learned how to write a Jet job that transform data and send them to a Feast feature server.
```
Comment (oliverhowell) [inline docs/modules/integrate/pages/streaming-features-with-feast.adoc:352]:
```suggestion
You also learned how to write a Jet job that transforms data and sends it to a Feast feature server.
```

**Review by oliverhowell — CHANGES_REQUESTED** [review]
Minor changes, sorry for delay

File: docs/modules/integrate/pages/streaming-features-with-feast.adoc (hunk 1 above)
Comment (yuce) [reply docs/modules/integrate/pages/streaming-features-with-feast.adoc]: Fixed at: https://github.com/hazelcast/hz-docs/pull/1201/commits/28518614a3e51945b93714ec0c2511469a6b4c61

File: docs/modules/integrate/pages/streaming-features-with-feast.adoc (hunk 2 above)
Comment (yuce) [reply docs/modules/integrate/pages/streaming-features-with-feast.adoc]: Fixed at: https://github.com/hazelcast/hz-docs/pull/1201/commits/28518614a3e51945b93714ec0c2511469a6b4c61

File: docs/modules/integrate/pages/streaming-features-with-feast.adoc (hunk 3 above)
Comment (yuce) [reply docs/modules/integrate/pages/streaming-features-with-feast.adoc]: Fixed at: https://github.com/hazelcast/hz-docs/pull/1201/commits/28518614a3e51945b93714ec0c2511469a6b4c61

**Review by oliverhowell — APPROVED** [review]

**Pull request merged**